mne/connectivity/cfc.py

```diff
@@ -0,0 +1,393 @@
+""" Cross frequency coupling functions. """
+# Authors: Praveen Sripad <praveen.sripad@rwth-aachen.de>
+#          Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#          Christopher Holdgraf <choldgraf@berkeley.edu>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from numpy.random import shuffle
+
+from ..stats import fdr_correction
+from ..time_frequency import morlet
+from ..time_frequency.tfr import cwt
+from ..preprocessing.peak_finder import peak_finder
+from ..filter import band_pass_filter
+from ..parallel import parallel_func
+from ..utils import check_random_state, ProgressBar
+
+
+def _abs_cwt(x, W):
+    """ Aux function """
+    return np.abs(cwt(x, [W], use_fft=False).ravel()).astype(np.float)
```

Comment:
I don't think you should need the `.astype(np.float)` if the point is to convert from complex to float, the `np.abs` should return `float` already. Or are you upconverting e.g. from `complex64` to `float64` (instead of `float32`) as well?